mathics/core/symbols.py

```diff
@@ -706,7 +733,7 @@ def symbol_set(*symbols: Tuple[Symbol]) -> FrozenSet[Symbol]:
 # PredefineSymbol.
 
 SymbolFalse = PredefinedSymbol("System`False", value=False)
-SymbolList = PredefinedSymbol("System`List")
+SymbolList = PredefinedSymbol("System`List", value=list)
```

Comment:
I am not totally sure about this. SymbolList is something that can't be changed so it is a PredefinedSymbol in that respect.

But it doesn't have a value like True or False do. (In the past I wondered also about Null)

The other option, is to allow the value parameter be optional for Predefined.

Reply:
The `value` field refers to a native Python's equivalent. For ```Integer``` the conversion is its value (of type int). For ```System`True``` the conversion is the Python's  `bool` `True`. For ```System`List``` the conversion is the class `list`, which is not too bad. Otherwise, I don't see the need to make ```System`List``` a ```PredefinedSymbol```.

Reply:
The motivation for ``System`List``  being not redefinable has to do with confidence to create a ListExpression. If a user could change List then we would have to consult the current binding (or "definition") of `List` to see if it were okay to create a ListExpression.

Reply:
Exactly, ```System`List``` is always interpreted as something used as the head of a List expression. And in Python, a head of a list expression is the ```list``` identifier. Regular ```Symbol``` (not ```PredefinedSymbol```s) should not have a (Python native) *value*. In any case, again the problem is how the Mathics API is (not clearly) defined.